testonly/integration/clientserver.go

```diff
@@ -40,12 +40,15 @@ const (
 )
 
 var (
+	publicKeyFile      = "../testdata/log-rpc-server.pubkey.pem"
```

Comment:
I think @RJPercival added a comment here about using the constants of testonly/keys.go instead of reading files, but I can't find it anymore.  :/

Could you change it, please?